src/less/main.less

```diff
@@ -0,0 +1,190 @@
+@some-grey: #727272;
+@some-other-grey: #dfdfdf;
+
+.transition-all {
```

Comment:
This is a mixin for making all properties transtionable. we can turn off particular ones if wanted.

docs:
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property